pinot-core/src/main/java/org/apache/pinot/core/operator/transform/TransformBlockDataFetcher.java

```diff
@@ -19,11 +19,7 @@
 package org.apache.pinot.core.operator.transform;
```

Comment:
Why do we need to touch this class? This is for selection queries only

Reply:
Discussed offline